match/lib/match/nuke.rb

```diff
@@ -252,11 +252,13 @@ def nuke_it_now!
 
       self.encryption.encrypt_files if self.encryption
 
-      # Now we need to commit and push all this too
-      message = ["[fastlane]", "Nuked", "files", "for", type.to_s].join(" ")
-      self.storage.save_changes!(files_to_commit: [],
-                                 files_to_delete: files_to_delete,
-                                 custom_message: message)
+      if files_to_delete.count > 0
+        # Now we need to commit and push all this too, if needed
+        message = ["[fastlane]", "Nuked", "files", "for", type.to_s].join(" ")
+        self.storage.save_changes!(files_to_commit: [],
+                                   files_to_delete: files_to_delete,
+                                   custom_message: message)
+      end
```

Comment:
Thoughts on doing a `UI.important` or `UI.user_error!` if count is 0? Feel like this needs to probably be communicated to the user 😇

Reply:
I don't feel like this is **that** _important_ (and definitely not an error), because `match nuke` should be idempotent and error'ing is exactly what led to the user's complaint. 

Although I do agree we could communicate - there's never too much communication 😁 
I'm on the fence between `important` and `message`. I'll go with `message`, feel free to change if you feel the priority should be changed 🙈